src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styles = sourceString
+    .split('\n')
+    .filter((style) => style !== '' && style.trim() !== ';');
+
+  const convertedString = {};
+
+  for (const style of styles) {
+    const [key, value] = style
+      .split(':')
+      .map((item) => item.trim());
+
+    convertedString[key] = value
+      .split('')
+      .filter(char => char !== ';').join('').trim();
```

Comment:
try splitting initially by `;` instead of `\n` so you will avoid this replacement